src/backend/commands/collationcmds.c

```diff
@@ -653,10 +653,8 @@ pg_import_system_collations(PG_FUNCTION_ARGS)
 				continue;		/* ignore locales for client-only encodings */
 			if (enc == PG_SQL_ASCII)
 				continue;		/* C/POSIX are already in the catalog */
-
-			/* GPDB_12_MERGE_FIXME: Why do we have this extra condition in GPDB? */
 			if (enc != GetDatabaseEncoding())
```

Comment:
Maybe

```suggestion
            /*
             * Greenplum specific behavior: this function in Greenplum can only be called after a full cluster is
             * built, this is different from Postgres which might call this function during initdb. When reaching
             * here, it must be in a database session, we can just ignore the collations not match current database's
             * encoding because they cannot be used in this database.
             */
			if (enc != GetDatabaseEncoding())
```

This function are from Postgres and we have some gpdb specific modification, it is recommended to add some comments starting with `Greenplum specific behavior` or `CDB` so that when later doing merge, we can resolve conflicts easier.